app/org/sagebionetworks/bridge/hibernate/HibernateAccount.java

```diff
@@ -0,0 +1,251 @@
+        return createdOn;
+    }
+
+    /** @see #getCreatedOn */
+    public void setCreatedOn(Long createdOn) {
+        this.createdOn = createdOn;
+    }
+
+    /** Account health code. */
+    public String getHealthCode() {
+        return healthCode;
+    }
+
+    /** @see #getHealthCode */
+    public void setHealthCode(String healthCode) {
+        this.healthCode = healthCode;
+    }
+
+    /** Account health ID, which maps to health code. */
+    public String getHealthId() {
+        return healthId;
+    }
+
+    /** @see #getHealthId */
+    public void setHealthId(String healthId) {
+        this.healthId = healthId;
+    }
+
+    /** Epoch milliseconds when the account was last modified, including password change. */
+    public Long getModifiedOn() {
+        return modifiedOn;
+    }
+
+    /** @see #getModifiedOn */
+    public void setModifiedOn(Long modifiedOn) {
+        this.modifiedOn = modifiedOn;
+    }
+
+    /** User's first name (given name). */
+    public String getFirstName() {
+        return firstName;
+    }
+
+    /** @see #getFirstName */
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    /** User's last name (surname). */
+    public String getLastName() {
+        return lastName;
+    }
+
+    /** @see #getLastName */
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    /**
+     * The algorithm used to hash the password.
+     *
+     * @see PasswordAlgorithm
+     */
+    @Enumerated(EnumType.STRING)
+    public PasswordAlgorithm getPasswordAlgorithm() {
+        return passwordAlgorithm;
+    }
+
+    /** @see #getPasswordAlgorithm */
+    public void setPasswordAlgorithm(PasswordAlgorithm passwordAlgorithm) {
+        this.passwordAlgorithm = passwordAlgorithm;
+    }
+
+    /** The full password hash, as used by {@link PasswordAlgorithm} to decode it. */
+    public String getPasswordHash() {
+        return passwordHash;
+    }
+
+    /** @see #getPasswordHash */
+    public void setPasswordHash(String passwordHash) {
+        this.passwordHash = passwordHash;
+    }
+
+    /** Epoch milliseconds when the user last changed their password. */
+    public Long getPasswordModifiedOn() {
+        return passwordModifiedOn;
+    }
+
+    /** @see #getPasswordModifiedOn */
+    public void setPasswordModifiedOn(Long passwordModifiedOn) {
+        this.passwordModifiedOn = passwordModifiedOn;
+    }
+
+    /**
+     * Set of user roles (admin, developer, researcher, etc). Never returns null.
+     *
+     * @see Roles
+     */
+    @CollectionTable(name = "AccountRoles", joinColumns = @JoinColumn(name = "accountId", referencedColumnName = "id"))
+    @Column(name = "role")
+    @ElementCollection(fetch = FetchType.EAGER)
+    @Enumerated(EnumType.STRING)
+    public Set<Roles> getRoles() {
+        if (roles == null) {
+            roles = EnumSet.noneOf(Roles.class);
+        }
+        return roles;
+    }
```

Comment:
Not able to use @Embedded/@Embeddable for a set of Roles, or do you want them in a separate table?